hooks/publish_file.py

```diff
@@ -661,6 +676,17 @@ def get_publish_dependencies(self, settings, item):
 
         return dependencies
 
+    def get_publish_user(self, settings, item):
+        """
+        Get the user that will be associated with this publish.
+
+        :param settings: This plugin instance's configured settings
+        :param item: The item to determine the publish template for
+
+        :return: A user entity dictionary.
```

Comment:
or none if not defined

Reply:
is this always set? may be worth talking about when it potentially returns none?